frontend/views/Landing.vue

```diff
@@ -131,6 +131,7 @@
                 <router-link
                   to="/home"
                   tag="button"
+                  alt="search"
```

Comment:
Same as above if this is simple decorative marker. But if it's directly attached to the button, it should be fine.

Reply:
The title attribute is optional for links. i will stay with this.